internal/handlers/api.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/gin-contrib/gzip"
 	"github.com/gin-gonic/gin"
 	"github.com/vultisig/mobile-tss-lib/tss"
+	"gitlab.com/thorchain/midgard/openapi/generated/oapigen"
```

Comment:
why we import the generate oapi from midgard?

Reply:
Resolved